src/input/types.js

```diff
@@ -72,6 +72,8 @@ export type BaseInputPropsT<T> = {
   'aria-describedby': ?string,
   /** Defines styles for inputs that are grouped with other controls. */
   adjoined: AdjoinedT,
+  /** Determines if browser should provide value suggestions. */
+  autocomplete: boolean,
```

Comment:
shouldn't it be a string value? https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete

Reply:
thank you